client/components/navbar.js

```diff
@@ -109,7 +109,7 @@ class Navbarclass extends Component {
         <Container fluid>
```

Comment:
Could we pull this container into it's own component as well? Then maybe only have it appear on products so we can setup a 'landing page'?